cpu/riscv_common/periph/Kconfig

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_RISCV_COMMON_PERIPH
+    bool
+    default y
+    depends on TEST_KCONFIG && CPU_ARCH_RISCV
+    help
+        Common peripheral code for RISC-V architecture.
+
+if MODULE_RISCV_COMMON_PERIPH
+
+config MODULE_PERIPH_CLIC
+    bool
+    depends on HAS_PERIPH_CLIC
+    help
+        Core-Local interrupt controller driver.
+
+config MODULE_PERIPH_CORETIMER
+    bool
+    depends on HAS_PERIPH_CORETIMER
+    help
+        Low-level timer driver based on the Core-local Interrupt (CLINT)
+
+config MODULE_PERIPH_PLIC
+    bool
+    depends on HAS_PERIPH_PLIC
+    help
+        Platform-Level interrupt controller driver.
+
+
+# TODO: this modules are actually just artifacts from the way periph_init_%
+# modules are handled in Makefile. We need to define them to keep the list the
+# same for now. We should be able to remove them later on.
+config MODULE_PERIPH_INIT_CLIC
+    bool
+    default MODULE_PERIPH_CLIC
+
+config MODULE_PERIPH_INIT_CORETIMER
+    bool
+    default MODULE_PERIPH_CORETIMER
+
+config MODULE_PERIPH_INIT_PLIC
+    bool
+    default MODULE_PERIPH_PLIC
+
+endif # MODULE_RISCV_COMMON_PERIPH
```

Comment:
You could add them here as well https://github.com/RIOT-OS/RIOT/blob/7bfcda979b2d538f39bd6e1f5f0e1e0a42a74fac/makefiles/features_modules.inc.mk#L13-L18

Reply:
I ran into the same a decided against adding modules just for the sake of it hahaha

Reply:
Now added in c9dc338